cost/turbonomics/rightsize_databases_recommendations/aws/CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Changelog
 
+## v0.3
+
+- Added `VM Age` incident field.
+- Added `Pricing Model` incident field.
+- Added `URL` incident field.
+- Added `Actual Storage Amount` incident field.
+- Added `Actual Storage Amount % Utilization` incident field.
```

Comment:
Use "Current" instead of "Actual". They mean different things. Example: "Current Storage Amount % Utilization"